<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArmListItem</class>
 <widget class="QWidget" name="ArmListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lb_armListItem">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>80</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_armFinger">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>手指</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_armIcon">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>武器图标</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_armName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>武器名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_armValue">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>武器值</string>
   </property>
  </widget>
  <zorder>lb_armFinger</zorder>
  <zorder>lb_armIcon</zorder>
  <zorder>lb_armName</zorder>
  <zorder>lb_armValue</zorder>
  <zorder>lb_armListItem</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
